docs/sphinx/specification/cudaq.rst

```diff
@@ -1,26 +1,32 @@
 Language Specification 
 ***********************************
 
-NVIDIA CUDA Quantum is a programming model for heterogeneous 
-quantum-classical computing available in C++ and Python.
-As such, the model provides primitive types, concepts, syntax, and
-semantics that enable the integration of quantum processors with
-existing NVIDIA multi-GPU parallel programming models and architectures. 
+NVIDIA CUDA Quantum is a single-source programming model in C++ for heterogeneous 
+quantum-classical computing. As such, the model provides primitive types, concepts, syntax, and
+semantics that enable the integration of quantum processing units (QPUs) with
+existing NVIDIA multi-node, multi-GPU parallel programming models and architectures. 
+The language is designed for C++ with extensions to other languages provided as via appropriate bindings. 
 
-CUDA Quantum is a library-based language extension that defines quantum code as typed
-callables in standard C++. These callables are meant to be compiled with
-standard C++ compilers and run via quantum computer simulators or physical
-quantum architectures. When annotated with a unique quantum function
-attribute, these functions can be enhanced with standard circuit
-optimization and transformation passes. 
+CUDA Quantum implementations are intended to be enabled via library-based language 
+extensions that define quantum code as callables in the classical language being extended. 
+These callables are meant to be compiled with standard compilers and runtimes 
+and run via quantum computer simulators or available physical quantum architectures. The 
+language callables intended for quantum coprocessing must be annotated in some way for 
```

Comment:
"in some way" seems a little ambiguous. Can we just say they must be annotated as QPU callables to be more precise here?